go/oasis-node/cmd/common/flags/flags.go

```diff
@@ -71,11 +73,21 @@ func DebugTestEntity() bool {
 	return DebugDontBlameOasis() && viper.GetBool(CfgDebugTestEntity)
 }
 
-// Entity returns the set entity directory.
+// Entity returns the entity backing.
 func Entity() string {
 	return viper.GetString(cfgEntity)
 }
 
+// EntityFileDir returns the set entity directory.
```

Comment:
Isn't this the signer file dir? It could be used for non-entity things? As the other comment says I think these should be the `Signer`, `SignerFileDir` and `SignerLedgerAddress` respectively.